app/services/send_email_service/send_notify_email.rb

```diff
@@ -43,5 +43,8 @@ def undeliverable_failure?(error)
     return false unless error.is_a?(Notifications::Client::BadRequestError)
 
     error.message.end_with?("Not a valid email address")
+
+    # We have a hard limit in notify that cannot be bypassed here.
+    error.message.include?("Your message is too long.")
```

Comment:
It'd be good to use `start_with?` here to be consistent with the earlier approach.